cdk/tooling_environment/stacks/grafana/lambda_functions_dashboard.template.json

```diff
@@ -195,8 +292,9 @@
         {
           "database": "<<DATABASE_NAME>>",
           "datasource": "Amazon-Timestream",
+          "format": 0,
           "measure": "execution",
-          "rawQuery": "SELECT time, ${measure:csv} FROM $__database.$__table where $__timeFilter and resource_name in (${resource_name:singlequote}) and monitored_environment in (${monitored_env:singlequote})",
+          "rawQuery": "select time, ${measure:csv} from ( \r\nSELECT\r\n  monitored_environment\r\n, resource_name\r\n, 1 as execution\r\n,(sum(invocation) over (partition by lambda_function_request_id)) -1 as retry_attempts\r\n, min(failed) over (partition by lambda_function_request_id) as failed\r\n, max(succeeded) over (partition by lambda_function_request_id) as succeeded\r\n, sum(duration_ms) over (partition by lambda_function_request_id) as duration_ms\r\n, sum(billed_duration_ms) over (partition by lambda_function_request_id) as billed_duration_ms\r\n, sum(memory_size_mb) over (partition by lambda_function_request_id) as memory_size_mb\r\n, sum(GB_seconds) over (partition by lambda_function_request_id) as  GB_seconds\r\n, max(max_memory_used_mb) over (partition by lambda_function_request_id) as max_memory_used_mb\r\n, max(time) over (partition by lambda_function_request_id) as time, row_number() over (partition by lambda_function_request_id order by time desc) as rn\r\n  FROM $__database.$__table )t\r\n  where rn = 1 and  $__timeFilter and resource_name in (${resource_name:singlequote}) and monitored_environment in (${monitored_env:singlequote})\r\n",
```

Comment:
for readability:
![image](https://github.com/user-attachments/assets/edc7aeac-a0ae-4fb3-bd70-b0ff06815df2)